### PR TITLE
feat(cli): add release command to cli

### PR DESCRIPTION
**Pull request by joshblack**
Add `release` command as a counterpart to https://github.com/carbon-design-system/carbon/pull/3589

#### Changelog

**New**

- Add new release command for creating a `patch` release

**Changed**

**Removed**

Preview URL: https://ibm.box.com/s/7mbhgc4f2y05yjom55iac0w6mbarz8cd

### CODE REVIEW

**Review by abbeyhrt — APPROVED** [review]
As far as I can tell, this looks good! Yay for pretty releases! 🎉

File: packages/cli/src/commands/release.js [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+  const [latestTag] = tags;
+
+  if (!latestTag) {
+    throw new Error('Unable to find latest tag from git tags');
+  }
+
+  logger.stop();
+  logger.start(`Finding next available version for bump: ${bump}`);
+
+  const nextTag = semver.inc(latestTag, 'patch');
```
Comment (abbeyhrt) [inline packages/cli/src/commands/release.js]:
```suggestion
  const nextTag = semver.inc(latestTag, bump);
```

File: packages/cli/src/commands/release.js [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+  }
+
+  logger.stop();
+
+  await execa(
+    'yarn',
+    [
+      'lerna',
+      'version',
+      'patch',
```
Comment (abbeyhrt) [inline packages/cli/src/commands/release.js]:
```suggestion
      bump,
```

**Review by jnm2377 — APPROVED** [review]

**Review by tw15egan — APPROVED** [review]
Handy! 👍

**Pull request merged**